### DISCUSSION TITLE
Edit metadata

### DISCUSSION THREAD

**Discussion by cedvan** [navidrome/navidrome#2418] [Ideas]
Hi,

It would be great if we could edit file metadata from the web interface.

In this way, an additional application such as MusicBee or MediaMonkey would become optional.

**Comment by dweymouth** [comment]
This will be unlikely to be added since Deluan has currently designed Navidrome to have read-only access to the music directory, partly for security reasons. Personally, when I add new music I adjust the tags to my liking and do a ReplayGain scan using Deadbeef on linux (Foobar on Windows would be the equivalent) and then drop it in the music folder.

**Comment by cedvan** [comment]
Personally, I'm aiming to stop using fat clients to manage my music. This follows the course of current software evolution. A device (PC, phone, etc.) must be interchangeable, and functionality must remain as identical as possible. Web applications offer this flexibility.

When I add digital files, it's usually via Internet links. My server is therefore able to retrieve them on its own, without having to send the files via my local PC. This implies that Navidrome with metadata editing capability would be a real plus. Currently, only Ampache lets you edit tags from a browser. I'm forced to keep Ampache and its outdated interface on the side just for this purpose, which is a real shame.

**Comment by cedvan** [comment]
What's more, it's quite possible to make a mistake or omission during the initial addition. That's why being able to continually revisit tags makes for a better library.

**Comment by dweymouth** [comment]
Oh I definitely think it would be convenient to have, and there's a possibility it might get added eventually (I think it's more likely Navidrome will eventually have tag-editing capabilities than browse-by-filesystem capabilities for example).
